M17_spec.tex

```diff
@@ -1436,150 +1494,33 @@ \subsection{Encryption Types}
-
-\subsubsection{Message Digest Algorithm for Voice Streams}
-At the beginning of the transmission, a $digest$ byte array of size 16 is initialized with zeros. After every stream frame (starting at frame 0) an exclusive or (XOR) operation is performed over the contents of the $digest$ array and the frame's payload. The $digest$ array is then rotated left by 1 byte. The result shall be retained in the array.
-
-\begin{align*}
-	digest :=& digest \oplus payload \\
-	digest :=& rol(digest, 8)
-\end{align*}
-
-This process is repeated until there is no more data to transmit. In case there is any encryption enabled, the $payload$ input shall be the encrypted stream. This ensures the possibility of verification, even if the encryption details are not known to the receiving parties. Frame Numbers of the frames carrying the signature should follow a succession of $\{7FFC_{16}, 7FFD_{16}, 7FFE_{16}, FFFF_{16}\}$.
-\begin{quote}
-	\textbf{NOTE} The Frame Number's most significant bit of the last speech payload stream shall not be set, since it is not the last frame to be transmitted.
-\end{quote}
-
-\subsubsection{Signature Generation and Transmission}
-At the transmitter-side, the stream digest is signed with a 256-bit private key. The resulting 512-bit signature is split into 4 chunks and sent as additional payload at the end of the transmission. To keep the reassembled LSF data consistent, the LICH counter shall advance normally. The most significant bit of the Frame Number (signalling end of transmission) shall be set only in the last frame carrying the signature.
+\section{Stream Frames}
 
-\subsubsection{Signature Verification}
-At the receiver-side, the 512-bit signature is retrieved from the last 4 frames' contents, if the appropriate TYPE bit is set. The signature is then checked using a 512-bit public key.
-\begin{quote}
-	\textbf{NOTE} The verification process will work if and only if all the data is received successfully (without transmission errors or dropped frames).
-\end{quote}
+Stream Frames will contain chunked LSF contents (in the LICH field). The Stream Contents will include the incrementing 16-bit Frame Number, and 128 bits of payload data (unencrypted or encrypted).
 
 \section{Packet Mode}
 
 \subsection{Packet Mode LSF TYPE}
 
-The TYPE field only defines the stream/packet bit, called ``P/S'' in Table~\ref{tab:packet_lsf_type} and the Channel Access Number (CAN) bits. All other bits are reserved.
+The Packet Mode TYPE is similar to the Stream Mode TYPE. It has the payload subfield set to \texttt{0xF} and the Channel Access Number (CAN) bits set at the user specified value. A single Meta field is supported in the Link Setup and could be GNSS data, Extened Callsign data or a one section Text data. Unlike Stream Mode, Packet Mode does not support encryption or digital signatures. 
```

Comment:
s/Extened/Extended/

The main difference is that Packet Mode doesn't have encryption, so consider phrasing it like this:

> Unlike Stream Mode, Packet Mode does not support encryption or digital signatures. Therefore, bits 0-3 of byte 0 are reserved when the payload subfield is set to \texttt{0xF}. Each packet has a single Link Setup META field. Its contents can be any of those defined in the META Contents, except those that are encryption or digital signature related.